docs/source/tutorial_optimization.md

```diff
@@ -777,6 +776,88 @@ assert opt_along_path_minimization_data['enrichments'] == 9
 assert opt_along_path_adaptively_minimization_data['enrichments'] == 4
 ```
 
+## Adaptive trust-region optimization
+
+Moreover, we can use trust-region methods to extend the adaptive enrichment with
+respect to the underlying error estimator.
+In the trust-region method, we replace the global problem with local surrogates,
+compute local solutions to these surrogate problems, and enhance the surrogate models
+when we either are close to the border of the trust-region or when the model confidence
+decreases such that we require more data.
+In effect, this creates two loops: the outer iterates over global parameter space and
+constructs local trust-regions along with their corresponding surrogates, whereas the
+inner solves the local problems with BFGS.
+For a fixed parameter in the outer iteration {math}`\mu` and the current surrogate model
+{math}`J_r`, this approach takes the shape
+
+```{math}
+\min_{\mu + s \in \Delta} J_r(u_{\mu + s}, \mu + s),  \tag{\hat{P}_r}.
+```
+
+In the formulation of ({math}`\hat{P}_r`), {math}`\Delta \subseteq \mathcal{P}` is the
+trust-region with tolerance {math}`\tau > 0` for an error estimator of our choice {math}`e_r`
+in which the updated parameter {math}`\mu + s` satisfies
+
+```{math}
+e_r(\mu + s) < \tau.
+```
+
+In metric settings such as with the {math}`2`-norm, these trust-regions correspond to
+open balls of radius {math}`\tau`, however the typical estimators in this tutorial's
+setting create much more complex shapes.
+
+The adaptive component of this algorithm is the choice of the trust-radius and the enrichment.
+If a local optimization in the inner loop fails, we retry the surrogate problem with a
+smaller trust-region, thus limiting the use of the current model.
+In contrast, when we quickly converge close to the boundary of the trust-region, it is a
+reasonable assumption that the globally optimal parameter is outside of the current region,
+therefore allowing us to enlarge the radius to include more parameter options in the
+inner optimization problem.
```

Comment:
I do not understand this sentence, to be honest. Do you mean that we enlarge the radius by enrichment or you mean the delta that can be enlarged for the next inner loop?